### DISCUSSION TITLE
Account Blocked

### DISCUSSION THREAD

**Discussion by LeilaSchooley** [vercel/community#4413] [Account Issue] [answered]
### Question

I want to buy a paid plan but my account was blocked for no reason upon sign up.

Tried to sign up via my legitimate Github account with lots of history etc, and was met by an instant block.

As expected, Vercel support team have not been helpful in the slightest, mainly because it's been two weeks and I haven't even receieved a reply to my ticket.
Obviously they dont care about their users and have chosen to act in a scummy manner, that is evident, but it seems their service is the best for hosting next.js apps.

Anyone else have this problem and how to deal with it?

Thanks

**Comment by leerob** [comment, marked as answer]
Sorry again about this, your account should be ready to sign up now 🙏
